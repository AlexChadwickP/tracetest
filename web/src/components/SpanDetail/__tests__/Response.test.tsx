import {render} from '@testing-library/react';
import Response from '../components/Http/Response';

test('Response', () => {
  const {container} = render(
    <Response
      attributeList={[
        {key: 'http.response.token', value: '2309ei'},
        {key: 'http.status_code', value: '200'},
        {key: 'http.response.content', value: '21232'},
      ]}
    />
  );
  expect(container).toMatchSnapshot();
  expect(container.getElementsByClassName('ant-table-row').length).toBe(2);
});
