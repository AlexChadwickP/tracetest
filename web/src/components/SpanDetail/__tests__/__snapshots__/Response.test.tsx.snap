// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Response 1`] = `
<div>
  <div
    class="ant-table-wrapper sc-bdvvtL kLJHSf"
  >
    <div
      class="ant-spin-nested-loading"
    >
      <div
        class="ant-spin-container"
      >
        <div
          class="ant-table ant-table-small ant-table-bordered ant-table-layout-fixed"
        >
          <div
            class="ant-table-container"
          >
            <div
              class="ant-table-content"
            >
              <table
                style="table-layout: fixed;"
              >
                <colgroup>
                  <col
                    style="width: 50%;"
                  />
                  <col
                    style="width: 50%;"
                  />
                </colgroup>
                <thead
                  class="ant-table-thead"
                >
                  <tr>
                    <th
                      class="ant-table-cell ant-table-cell-ellipsis"
                      title="Key"
                    >
                      Key
                    </th>
                    <th
                      class="ant-table-cell ant-table-cell-ellipsis"
                      title="Value"
                    >
                      Value
                    </th>
                  </tr>
                </thead>
                <tbody
                  class="ant-table-tbody"
                >
                  <tr
                    class="ant-table-row ant-table-row-level-0"
                    data-row-key="http.response.token"
                  >
                    <td
                      class="ant-table-cell ant-table-cell-ellipsis"
                      title="http.response.token"
                    >
                      http.response.token
                    </td>
                    <td
                      class="ant-table-cell ant-table-cell-ellipsis"
                      title="2309ei"
                    >
                      2309ei
                    </td>
                  </tr>
                  <tr
                    class="ant-table-row ant-table-row-level-0"
                    data-row-key="http.response.content"
                  >
                    <td
                      class="ant-table-cell ant-table-cell-ellipsis"
                      title="http.response.content"
                    >
                      http.response.content
                    </td>
                    <td
                      class="ant-table-cell ant-table-cell-ellipsis"
                      title="21232"
                    >
                      21232
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
